frontend/src/hooks/useAnalytics.tsx

```diff
@@ -38,8 +52,19 @@ export default function useChannel() {
             const res = await axios.get(`/api/channels`);
             setChannels(res.data);
         })();
+
+        (async () => {
+            const res = await axios.get(`/api/metrics`);
+            setMetrics(res.data);
+        })();
     }, []);
 
+    // if(!appUser)
+    // {
+    //     return <Navigate to={"/logout"}/>;
+    // }
```

Comment:
Kann das weg?

Reply:
Das ist tatsächlich neuer Code, der noch nicht in Verwendung ist. Die Umstrukturierung wollte ich aber später machen.